Mathlib/Data/Matrix/Notation.lean

```diff
@@ -280,32 +280,32 @@ section VecMul
 variable [NonUnitalNonAssocSemiring α]
 
 @[simp]
-theorem empty_vecMul (v : Fin 0 → α) (B : Matrix (Fin 0) o' α) : vecMul v B = 0 :=
+theorem empty_vecMul (v : Fin 0 → α) (B : Matrix (Fin 0) o' α) : v ᵥ* B = 0 :=
   rfl
 #align matrix.empty_vec_mul Matrix.empty_vecMul
 
 @[simp]
-theorem vecMul_empty [Fintype n'] (v : n' → α) (B : Matrix n' (Fin 0) α) : vecMul v B = ![] :=
+theorem vecMul_empty [Fintype n'] (v : n' → α) (B : Matrix n' (Fin 0) α) : v ᵥ* B = ![] :=
   empty_eq _
 #align matrix.vec_mul_empty Matrix.vecMul_empty
 
 @[simp]
 theorem cons_vecMul (x : α) (v : Fin n → α) (B : Fin n.succ → o' → α) :
-    vecMul (vecCons x v) (of B) = x • vecHead B + vecMul v (of <| vecTail B) := by
+    vecCons x v ᵥ* (of B) = x • vecHead B + v ᵥ* (of <| vecTail B) := by
```

Comment:
Now that the precedence has changed, this can become
```suggestion
    vecCons x v ᵥ* of B = x • vecHead B + v ᵥ* of (vecTail B) := by
```
similarly in many place where there are `()`s (it looks like you already cleaned up one of the arguments, thanks!)

Reply:
Yeah, I was cleaning almost all the places, but `of` without `()` was giving me the ick.